src/neuroconv/datainterfaces/ecephys/requirements.txt

```diff
@@ -1,2 +1,3 @@
-spikeinterface>=0.97,<0.98
+spikeinterface>=0.98.1
+neo<=0.11.1
```

Comment:
why are you forcing neo to not use the latest release?

Reply:
Yes, it should be allowed.

There was a problem with the latest release and spikeinterface extractors that I just fixed:

https://github.com/SpikeInterface/spikeinterface/pull/1848

(That's where the constrain was coming from)

There will be a new release of Spikeinterface with the fix above within this week.

Reply:
All right, this is corrected now, this is ready.